__tests__/api.test.js

```diff
@@ -122,4 +122,43 @@ describe("app", () => {
         });
     });
   });
+  describe("GET /api/articles/:article_id/comments", () => {
```

Comment:
I'd like to see a test for what happens if you get an empty array back as there is no comments.
This would still be aa 200 success.